### PR TITLE
- Release prep

### PR DESCRIPTION
**Pull request by seriva**
- Updated changelog.
- Update for versioning.

### CODE REVIEW

File: CHANGELOG-0.4.md [hunk 1]
```diff
@@ -1,10 +1,14 @@
 # Changelog 0.4
 
-## [0.4.2] 2019-11-TOCHANGE
+## [0.4.2] 2019-11-15
 
 ### Added
 
-- Offline upgrade for K8n and Docker
+- online/offline upgrade for K8n and Docker
```
Comment (toszo) [inline CHANGELOG-0.4.md]: it is "K8s" :)

File: CHANGELOG-0.4.md [hunk 2]
```diff
@@ -1,10 +1,14 @@
 # Changelog 0.4
 
-## [0.4.2] 2019-11-TOCHANGE
+## [0.4.2] 2019-11-15
```
Comment (toszo) [inline CHANGELOG-0.4.md]: Create draft release since this date will be different.

File: CHANGELOG-0.4.md (hunk 1 above)
Comment (seriva) [reply CHANGELOG-0.4.md]: Fixed

File: CHANGELOG-0.4.md (hunk 2 above)
Comment (seriva) [reply CHANGELOG-0.4.md]: Fixed

File: core/src/epicli/cli/version.py [hunk 3]
```diff
@@ -1 +1,2 @@
-VERSION = '0.4.2'
\ No newline at end of file
+import os
+VERSION = open(os.path.join( os.path.dirname(__file__), 'version.txt.py'), 'r').read()
```
Comment (lukurde) [inline core/src/epicli/cli/version.py]: ugly space
```suggestion
VERSION = open(os.path.join(os.path.dirname(__file__), 'version.txt.py'), 'r').read()
```

**Pull request merged**